middlewares/taskRequests.js

```diff
@@ -1,12 +1,17 @@
 const { SOMETHING_WENT_WRONG } = require("../constants/errorMessages");
 const dataAccess = require("../services/dataAccessLayer");
+const { TASK_REQUEST_UPDATE_ACTION } = require("../constants/taskRequests");
 /**
  * Validates user id for task request
  *
  * @param userId { string }: user id of the user
  */
 async function validateUser(req, res, next) {
   try {
+    const { action } = req.query;
+    if (action === TASK_REQUEST_UPDATE_ACTION.REJECT) {
+      return next();
+    }
```

Comment:
- why are we skipping validation here?

Reply:
UserId is not required for rejecting a task request. So no validation is required.

Reply:
- I am a little confused about our approach here, can you please help me briefly understand the request flow or guide me to a design doc?

Reply:
- This function is a validation middleware which was previously used for the route PATCH /taskRequests/approve and this route had only the option to approve a task request. 
- In this PR we are modifying the api to have the option to both approve and reject a task request
- When rejecting a task request there's no point in rejecting it for a particular user/requestor so user id is not mandatory and validation is skipped.